src/som/vm/Shell.java

```diff
@@ -94,7 +96,7 @@ public SAbstractObject start() {
 
         // If success
         if (myClass != null) {
-          currentFrame = interpreter.getFrame();
+          currentFrame = frame;
```

Comment:
Hm, I am not entirely sure whether these two are equivalent.
The frame may have changed in the interpreter.

The question is whether we want/need to support this.

Is there a use case where we want the loading of a class to be able to change the frame?

We are also in the loop, so, on the next iteration, the code may have returned from another frame? Is that possible? I am not sure. The code is not explicit about why that's needed, is it?

Reply:
I hadn't really paid attention to the shell mode to be honest. Now that I did, I tried to delete the silly hot fixes I added to make it compile. It should behave now the same way as it was behaving in som-java / TruffleSOM. This could be improved but I am not sure I want to spend time on this for now.